translations/setup.tex/ua.po

```diff
@@ -124,6 +127,60 @@ msgid ""
 "\\end{enumerate}\n"
 "\n"
 msgstr ""
+"\\begin{enumerate}\n"
+"  \\item Виберіть сценарій з Книги Місій.\n"
+"    Для першої гри ми рекомендуємо обрати сценарій \"Прекрасний новий світ\" (див. стор. 7 в Кинзі Місій).\n"
+"  \\item Виберіть свою фракцію з тих, що є в наявності.\n"
+" \\item Виберіть одного з Героїв вашої фракції в якості Головного Героя.\n"
+"    Кожна фракція має принаймні одну двосторонню картку героя, на кожній стороні якої зображений герой.\n"
+"  \\item Візьміть наступні компоненти, що належать вашій фракції:\n"
+"  \\begin{itemize}\n"
+"    \\item[a)]1 × Двостороння карта героя (на стороні обраного героя)\n"
+"    \\item[b)]2 × Фігурки героїв\n"
+"    \\item[c)]7 × Тайлів Міських Будівель\n"
+"    \\item[d)]1 × Планшет Міста\n"
+"    \\item[e)]7 × Двосторонніх Карток Підрозділів\n"
+"    \\item[f)]3 × Картки Спеціалізації Героя(обраного Героя)\n"
+"    \\item[g)]1 × Карту Здібностей Героя(обраного Героя)\n"
+"    \\item[h)]20 × Кубиків Фракції\n"
+"    \\item[i)]1 × Жетон Будівництва\n"
+"    \\item[j)]1 × Жетон Населення\n"
```

Comment:
I don't know how Ukrainian works, but in Russian we at first started with the same capitalization as in English text, but then decided that it looks better when not everything is capitalized; first several chapters are not updated though yet, we'll update them after finishing the translation

Reply:
I'm just following what is written in the Introduction of the Rulebook: "In this rulebook, game and component terms are Capitalized."
As for me, this makes the process of preparing for the game a little more convenient, if not, I can change it in the text

Reply:
Sure, whatever works better for you!

Reply:
And 1 more question about translation, how did you translate the terms: Visitable and Revisitable? 
Because I can't think of a better translation than: Одноразовые and Многоразовые

Reply:
This is exactly what Polish version has. Jednorazowe and Wielorazowe.